APITest/APITest/Controllers/EmployeeController.cs

```diff
@@ -1,23 +1,77 @@
-﻿using RestSharp;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
+using RestSharp;
+using APITest.Constants;
 
 namespace APITest.Controllers
 {
-    public class EmployeeController : BaseController
+    public class EmployeeController
     {
-        private const string GetEmployeeUrl = "/employees";
-        private const string GetEmployeeByIdUrl = "/employees/{0}";
+        private async Task AddCookiesInRequest(RestRequest request)
+        {
+            var response = await GetAllEmployeesAsync();
+
+            request.AddCookie("PHPSESSID", response.Cookies[0].Value);
+        }
+
+        public async Task<IRestResponse> GetAllEmployeesAsync()
+        {
+            var client = new RestClient(ConfigConstants.GetAllEmployeesURL);
+
+            var request = new RestRequest(ConfigConstants.GetAllEmployeesURL, Method.GET);
+
+            return await Task.Run(() => client.Get(request));
+        }
+
+        public async Task<IRestResponse> GetEmployeeByIdAsync(uint employeeId)
```

Comment:
check type? is it crucial to use unsigned one?

Reply:
Because it is making a narrow range of potentially wrong values. Traditionally IDs used only unsigned values. This API too